File: Ruby/day5.rb
t = gets.to_i
(1..t).each do |i|
    a, b, n = gets.split.map { |s| s.to_i }
    p = 1
    r = [a + b * p]
    (2..n).each do |j|
        p *= 2
        r.push(r[-1] + b * p)
    end
    puts r.join(" ")
end

File: Python/day5.py
t = int(raw_input())
for i in range(t):
    a, b, n = map(int, raw_input().split())
    p = 1;
    r = [a + b * p]
    for j in range(1, n):
        p *= 2
        r.append(r[-1] + b * p)
    print " ".join(map(str, r))
